main.css

```diff
@@ -0,0 +1,9 @@
+body {
+    background-color: red;
+    color: white;
```

Comment:
This is completely illegible. Please invert it or something. I know you love red, but maybe not the background? ❤️